env/env.js

```diff
@@ -68,6 +68,14 @@ const development = removeNullValues({
   ...developmentRaw,
 })
 
+const developmentRemote = removeNullValues({
```

Comment:
Why don't you just make this dev?  Individuals can override it using `developmentRaw`?